runtime/common/src/migrations.rs

```diff
@@ -454,6 +458,73 @@ where
     }
 }
 
+pub struct RegistrarReserveToHoldMigration<T>(pub PhantomData<T>);
+impl<T> Migration for RegistrarReserveToHoldMigration<T>
+where
+    T: pallet_registrar::Config,
+    T: pallet_balances::Config,
+    <T as pallet_balances::Config>::RuntimeHoldReason: From<pallet_registrar::HoldReason>,
+    <T as pallet_balances::Config>::Balance: From<
+        <<T as pallet_registrar::Config>::Currency as frame_support::traits::fungible::Inspect<
+            T::AccountId,
+        >>::Balance,
+    >,
+    <T as pallet_balances::Config>::Balance: From<u128>,
+{
+    fn friendly_name(&self) -> &str {
```

Comment:
do we know how many of these we have? did we try with try-runtime what is the weight used in, e.g., dancebox¿?

Reply:
```
Flashbox consumed weight: Weight(ref_time: 19450000000, proof_size: 0)
Dancebox consumed weight: Weight(ref_time: 11200000000, proof_size: 0)
```

Reply:
that's quite little, cool